python_modules/libraries/dagster-tableau/dagster_tableau/translator.py

```diff
@@ -109,6 +109,7 @@ def namespace(cls) -> str:
 class TableauMetadataSet(NamespacedMetadataSet):
     id: Optional[str] = None
     workbook_id: Optional[str] = None
+    has_extracts: Optional[bool] = None
```

Comment:
optional bool is a pretty freaky data type -- can we just have this default to False or something?

Reply:
I created a subclass in [db6ebf6](https://github.com/dagster-io/dagster/pull/29679/commits/db6ebf662d767272043fbdcc8e49808e5c3ae17a), and make this default to False.

For the context, views and data sources were sharing the same metadata set, but `has_extracts` is n/a for views - setting a default to False was automatically adding `{"dagster-tableau/has_extracts": False}` to the metadata of all views. Using the subclass fixes the issue.